CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHRRecoveryStepViewController.swift

```diff
@@ -34,6 +34,33 @@
 
 import UIKit
 
+extension CRFHRRecoveryStep : RSDStepViewControllerVendor {
+    
+    func instantiateViewController(with parent: RSDPathComponent?) -> (UIViewController & RSDStepController)? {
+        let vc = CRFHRRecoveryStepViewController(step: self, parent: parent)
+        return vc
+    }
+    
+    func resultRange(from taskResult: RSDTaskResult) -> (min: Double, max: Double)? {
+        guard let answerResult = self.answerResult(from: taskResult),
+            let answer = (answerResult.value as? RSDJSONNumber)?.jsonNumber()?.doubleValue
+            else {
+                return nil
+        }
+        return (answer * 0.95, answer * 1.05)
```

Comment:
This to be replaced with confidence intervals from the validation study once analysis from @larssono / @itismeghasyam is complete